lessons/advanced/gen-stage-1.md

```diff
@@ -0,0 +1,368 @@
+---
```

Comment:
My idea for the structure here was functional super basic example --> working basic clone of delayed_job --> explain the technical innards in a (hopefully) friendlier way than the docs.